FE/src/components/header/searchBar/index.tsx

```diff
@@ -1,15 +1,43 @@
-import React, { useState } from 'react';
+import React, { useReducer, useState } from 'react';
+
 import * as Styled from 'components/header/searchBar/searchBar.style';
+import Line from 'components/Icons/Line';
 import SearchBarItem from 'components/header/searchBar/searchBarItem';
 import SearchIcon from 'components/Icons/SearchIcon';
+import { ActionType, reducer } from 'components/header/searchBar/contentReducer';
+import priceData from 'components/mock/priceData';
+import PriceModal from './priceModal';
+import { customStyles } from './priceModal/PriceModalInfo.style';
+
+const initState = { checkIn: '날짜 입력', checkOut: '날짜 입력', price: '금액대 설정', guest: '게스트 추가' };
 
 function SearchBar() {
+  const [state, dispatch] = useReducer(reducer, initState);
+  const [clickTitle, setClickTitle] = useState('');
+
+  const clickItem = (keyData, valueData) => () => {
+    setClickTitle(keyData);
+    dispatch({
+      type: ActionType.SET_CONTENTS,
+      payload: { key: keyData, value: valueData },
+    });
```

Comment:
```suggestion
      type: ActionType.SET_CONTENTS,
      payload: { key: keyData, value: valueData },
    });
```

해당 부분 액션 생성함수로 빼면 의미 전달이 더 잘될 것 같습니다! (함수 이름은 적당한 걸로 골라주세요!)
```
const setContents = ({key, value}) => {
 return {
      type: ActionType.SET_CONTENTS,
      payload: { key, value },
    }
}


dispatch(setContents({key: keyData, value: valueData });
```

Reply:
^0^)b 넵! 반영해보겠습니다!